content/docs/studio/user-guide/teams.md

```diff
@@ -1,15 +1,25 @@
+<admon>
+
+**We have renamed Views to Projects in Iterative Studio.**
+
+Accordingly, _Views dashboard_ is now called _Projects dashboard_; _View
+settings_ are now called _Project settings_; and so on.
```

Comment:
Possibly x4 (not sure on this one since it does have lots of mentions of _Projects_.